modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/ImportObjectDefinitionMVCActionCommand.java

```diff
@@ -117,44 +118,97 @@ private void _importObjectDefinition(ActionRequest actionRequest)
 		UploadPortletRequest uploadPortletRequest =
 			_portal.getUploadPortletRequest(actionRequest);
 
+		JSONArray jsonArray = _jsonFactory.createJSONArray();
+
 		String objectDefinitionJSON = FileUtil.read(
 			uploadPortletRequest.getFile("objectDefinitionJSON"));
 
-		JSONObject objectDefinitionJSONObject = _jsonFactory.createJSONObject(
-			objectDefinitionJSON);
+		boolean listImport = Validator.isNull(objectDefinitionJSON);
```

Comment:
@G-Sa this variable name does not sound good. 

Do we really need to create this variable? Can't we do `Validator.isNull(objectDefinitionJSON)` where it's needed? 

If we really need it, I suggest something like `hasMoreThanOneObjectDefinition`.

Reply:
We can, but since we would have to call it a couple of times i thought it would fall into the guideline for assigning to variables values that are queried more than once

Reply:
@G-Sa please rename it as suggested

Reply:
Done